lib/screens/explore_screens/vote_screens/delegate/delegators_tab/interactor/usecase/delegator_load_data_usecase.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:async/async.dart';
+import 'package:seeds/datasource/local/settings_storage.dart';
+import 'package:seeds/datasource/remote/api/eos_repository.dart';
+import 'package:seeds/datasource/remote/api/members_repository.dart';
+import 'package:seeds/datasource/remote/api/proposals_repository.dart';
+import 'package:seeds/datasource/remote/model/delegator_model.dart';
+
+class DelegatorLoadDataUseCase {
+  final ProposalsRepository _proposalsRepository = ProposalsRepository();
+  final MembersRepository _membersRepository = MembersRepository();
+
+  Future<List<Result>?> run() async {
+    final account = settingsStorage.accountName;
+    final result = await _proposalsRepository.getDelegator( account, EosRepository.voiceScopeCampaign);
+
+    if (result.isError) {
+      return null;
```

Comment:
Same why return null for error results - don't we want to just return error

Reply:
LoadDelegatorDataUseCase

Reply:
I see why he is doing that. 

Gaby, look at how i am doing it in invites manager use case.

Reply:
I copied from your code lol

Reply:
usecase now returns a list of results, also Rename usecase and mapper as suggested.